test.json

```diff
@@ -0,0 +1 @@
+{"event_info": {"name": "test", "event date": "2020-02-02 20:21:22", "event description": "A very progressive test run at                 Sunnyside Daycare's Butterfly Room.", "venue name": "Venue 1", "venue description": "foo"}, "measurement_info": [{"sensor name": "test", "timestamp": "2020-04-28 06:12:11.415791", "values": {"lefts": 30}}]}
```

Comment:
This file doesn't seem to be used.

Reply:
I am sorry but I have no idea how this file came into being. I am deleting this file.